libs/langchain-community/src/stores/message/cassandra.ts

```diff
@@ -131,6 +134,8 @@ export class CassandraChatMessageHistory extends BaseListChatMessageHistory {
       return;
     }
 
+    this.client = await CassandraStore.getClient(this.options);
```

Comment:
I think this is fine for now, another option would be to put this under `utils/`